doc/rst/technotes/extern.rst

```diff
@@ -650,15 +746,20 @@ code using an ``extern block`` as follows:
     ....
   }
 
-Such 'extern { }' block statements add the top-level C statements to
+Such ``extern { }`` block statements add the top-level C statements to
 the enclosing Chapel module.  This is similar to what one might do
 manually using the extern declarations (as described above), but can
 save a lot of labor for a large API.  Moreover, using an inline extern
 block permits you to write C declarations directly within Chapel
-without having to create distinct C files.  Using an extern block in
-this way also injects an implicit ``public use SysCTypes;`` statement
-into the scope of the extern block to make standard C types that the
-extern block is likely to require available.
+without having to create distinct C files.
+
+An ``extern { }`` also adds ``private use`` statements for the following
```

Comment:
I might keep the "extern block" label in addition to having the syntax, as it would help people know what to search for if they wanted more information

Reply:
This is in a section discussing *extern block* and that's in the title etc. So I'm not sure it's worth making it more complex. Did you have something specific in mind?

Reply:
Ah, okay.  And I also missed that the previous paragraph already handled it, so I'm willing to yield on this